src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -94,11 +94,8 @@
     public static final String TRANSMISSION_DISEQUILIBRIUM_KEY =    "TDT";
     public static final String VARIANT_TYPE_KEY =                   "VariantType";
     public static final String VQS_LOD_KEY =                        "VQSLOD";
-    public static final String OXOG_ALT_F1R2_KEY =                  "ALT_F1R2";
-    public static final String OXOG_ALT_F2R1_KEY =                  "ALT_F2R1";
-    public static final String OXOG_REF_F1R2_KEY =                  "REF_F1R2";
-    public static final String OXOG_REF_F2R1_KEY =                  "REF_F2R1";
-    public static final String OXOG_FRACTION_KEY =                  "FOXOG";
+    public static final String F1R2_KEY =                           "F1R2";
```

Comment:
What if I make it `VCFHeaderLineCount.R` instead of `VCFHeaderLineCount.A` and unify the ref's and alts' annotations under the single name `F1R2`?  This would be consistent with other GATK annotations.